code/modules/clothing/shoes/magboots.dm

```diff
@@ -91,3 +92,12 @@
 	if(item_flags & NOSLIP)
 		state = "enabled"
 	user << "Its mag-pulse traction system appears to be [state]."
+	
+/obj/item/clothing/shoes/magboots/cheap
+	name = "budget magboots"
+	desc = "Cheap magnetic boots, often used during extravehicular activity to ensure the user remains safely attached to the vehicle. Unlike the high quality variant, these magboots are uncomfortably small, and are not designed to fit non-humanoid feet."
+	magslowdown = 6
+	overshoes = 0
+	species_restricted = list("exclude","Unathi","Tajara","Vox","Vaurca","Vaurca Breeder")
+	//These are specifically made for mining. The logic here is that generally anyone with a whitelist isn't a fucking idiot and won't fall to their death from walking off a cliff.

```

Comment:
Magboots in game do not have a species restriction, why those would have based on such ooc concern? Also, I am sure that being whitelisted does not really have any relation to being robust or avoiding holes.